ReadersAndroidJetpack/app/src/main/java/com/kontinua/readersandroidjetpack/views/PDFView.kt

```diff
@@ -11,6 +11,7 @@ import androidx.compose.ui.Modifier
 import androidx.compose.ui.platform.LocalContext
 import androidx.compose.ui.viewinterop.AndroidView
 import androidx.lifecycle.viewmodel.compose.viewModel
+import android.util.Log
```

Comment:
left over from manual debugging? doesn't seem to be used by the other code added in this changeset.